Crypt/GPG.php

```diff
@@ -1392,6 +1392,27 @@ public function handleImportKeyStatus($line, array &$result)
     }
 
     // }}}
+
+    /**
+     * Tell if there are encryption keys registered
+     *
+     * @return boolean True if the data shall be encrypted
+     */
+    public function hasEncryptKeys()
+    {
+        return count($this->encryptKeys);
```

Comment:
I'd prefer an explicit boolean statement for the return value like `count($a) > 0` rather than relying on type coercion.